config/nrfconnect/chip-module/Kconfig

```diff
@@ -249,7 +245,7 @@ endif # CHIP_FACTORY_DATA_BUILD
 # See config/zephyr/Kconfig for full definition
 config CHIP_FACTORY_RESET_ERASE_NVS
 	bool
-	default y
+	default y if !CHIP_CRYPTO_PSA # For now erasing whole NVS sector is not supported for PSA Crypto
```

Comment:
If this is not supported and cannot be enabled by the user, probably we should add `depends on !CHIP_CRYPTO_PSA` to the `CHIP_FACTORY_RESET_ERASE_NVS` declaration (not default override).

Reply:
Btw. why we even redefine this Kconfig here? If we want to set the default to y we should not use the type and move it to Kconfig.defaults. Any thoughts?